File: hammer-gui/src/main/java/dev/vini2003/hammer/gui/HGUIClient.java
package dev.vini2003.hammer.gui;

import dev.vini2003.hammer.gui.registry.client.HGUIEvents;
import net.fabricmc.api.ClientModInitializer;
import net.fabricmc.api.ModInitializer;
import org.jetbrains.annotations.ApiStatus;

@ApiStatus.Internal
public class HGUIClient implements ClientModInitializer {
	@Override
	public void onInitializeClient() {
		HGUIEvents.init();
	}
}

File: hammer-gui/src/main/kotlin/dev/vini2003/hammer/gui/HGUIClient.kt
package dev.vini2003.hammer.gui

import dev.vini2003.hammer.gui.registry.client.HGUIEvents
import dev.vini2003.hammer.gui.registry.client.HGUIKeyBinds
import net.fabricmc.api.ClientModInitializer

object HGUIClient : ClientModInitializer {
	override fun onInitializeClient() {
		HGUIEvents.init()
		HGUIKeyBinds.init()
	}
}